src/contracts/token/Eigen.sol

```diff
@@ -193,4 +201,12 @@ contract Eigen is OwnableUpgradeable, ERC20VotesUpgradeable {
     function CLOCK_MODE() public pure override returns (string memory) {
         return "mode=timestamp";
     }
+
+    /**
+     * @notice Returns the bEIGEN token
+     * @return The bEIGEN token
+     */
+    function beigen() public view returns (IERC20) {
+        return bEIGEN;
+    }
```

Comment:
We already have a method `Eigen.bEIGEN`

Why add another one here?

Reply:
right, just recall solidity compiler auto adds getters